src/pages/library/DataBrowserPreview.js

```diff
@@ -143,15 +144,15 @@ const DataBrowserPreview = ({ id }) => {
             icon('times', { size: 30 })
           ])
         ]),
-        dataset.access === datasetAccessTypes.CONTROLLED && div({ style: { display: 'flex', flexDirection: 'row', backgroundColor: 'white', fontSize: '1.1rem', lineHeight: '1.7rem', padding: '20px 30px 25px', width: 'fit-content', margin: 'auto' } }, [
+        access === datasetAccessTypes.CONTROLLED && div({ style: { display: 'flex', flexDirection: 'row', backgroundColor: 'white', fontSize: '1.1rem', lineHeight: '1.7rem', padding: '20px 30px 25px', width: 'fit-content', margin: 'auto' } }, [
           h(RequestDatasetAccessModal, {
             datasets: [dataset],
             onDismiss: () => {
               Nav.goToPath('library-details', { id: Nav.getCurrentRoute().params.id })
             }
           })
         ]),
-        dataset.access === datasetAccessTypes.GRANTED && h(GroupedSelect, {
+        access === datasetAccessTypes.GRANTED && h(GroupedSelect, {
```

Comment:
I wonder if there's a way to group all enum options together. It would be nice if it was apparent to a reader that there's a single property (`access`) which is controlling single UI element here. As written a closer read is needed to know that you can't have two different `access` types at once Could a `switch` be used instead?

Reply:
It can't quite, cause CONTROLLED has two options (depending on some other variables). This one can maybe be a switch though, I haven't looked at preview that much.

Reply:
Yeah, I think here it can be a switch. Will change.